msteams-platform/webhooks-and-connectors/how-to/connectors-using.md

```diff
@@ -98,55 +136,17 @@ You can also use this JSON to create cards containing rich inputs, such as text
 }
 ```
 
-This message produces the following card in the channel:
-
-![Screenshot of a Connector card](~/assets/images/connectors/connector_message.png)
+This message provides the following card in the channel:
 
-## Creating actionable messages
+![Screenshot of a connector card](~/assets/images/connectorcard.png)
 
-The example in the preceding section includes three visible buttons on the card. Each button is defined in the `potentialAction` property of the message by using `ActionCard` actions, each containing an input type: a text field, a date picker, or a multi-choice list. Each `ActionCard` action has an associated action, for example `HttpPOST`.
+## Send messages using cURL and PowerShell
 
-Connector cards support three types of actions:
+# [cURL](#tab/cURL)
 
-- `ActionCard` Presents one or more input types and associated actions
-- `HttpPOST` Sends a POST request to a URL
-- `OpenUri` Opens a URI in a separate browser or app; optionally targets different URIs based on operating systems
-
-The `ActionCard` action supports three input types:
-
-- `TextInput` A single-line or multiline text field with an optional length limit
-- `DateInput` A date selector with an optional time selector
-- `MultichoiceInput` A enumerated list of choices offering either a single selection or multiple selections
-
-`MultichoiceInput` supports a `style` property that controls whether the list initially appears fully expanded. The default value of `style` depends on the value of `isMultiSelect`.
-
-| `isMultiSelect` | `style` default |
-| --- | --- |
-| `false` or not specified | `compact` |
-| `true` | `expanded` |
+**To post a message in the webhook with cURL**
 
-If you want a multiselect list initially displayed in the compact style, you must specify both `"isMultiSelect": true` and `"style": true`.
-
-For more information on Connector card actions, see **[Actions]**(/outlook/actionable-messages/card-reference#actions) in the actionable message card reference.
-
-> [!NOTE]
-> Specifying `compact` for the `style` property in Microsoft Teams is the same as specifying `normal` for the `style` property in Microsoft Outlook.
-> 
-> For the HttpPOST action, the bearer token is included with the requests. This token includes the Azure AD identity of the Office 365 user who took the action.
-
-## Setting up a custom incoming webhook
-
-Follow these steps to see how to send a simple card to a Connector:
-
-1. In Microsoft Teams, choose **More options** (**&#8943;**) next to the channel name and then choose **Connectors**.
-1. Scroll through the list of Connectors to **Incoming Webhook**, and choose **Add**.
-1. Enter a name for the webhook, upload an image to associate with data from the webhook, and choose **Create**.
```

Comment:
This seems repeated from Create incoming webhook page. Ref: https://review.docs.microsoft.com/en-us/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook?branch=pr-en-us-3657#add-an-incoming-webhook-to-a-teams-channel